warehouse/warehouse.go

```diff
@@ -725,6 +729,59 @@ func (wh *HandleT) runUploadJobAllocator() {
 	}
 }
 
+func (wh *HandleT) uploadStatusTrack() {
+	for {
+		time.Sleep(uploadStatusTrackFrequency)
```

Comment:
put this sleep at the end of function instead? just to get atleast one stat when pod restarts

Reply:
done